library/src/utils/position/groupByOverlappingTimesRange.ts

```diff
@@ -1,43 +1,39 @@
 import { BaseTask } from '../../types/baseTask';
 
-function groupByOverlappingTimesRange<T extends BaseTask>(
-  test: (T & {
-    ref: (node: HTMLElement | null) => void;
-    style: React.CSSProperties;
-  })[],
-) {
-  const sortedTasks = [...test].sort(
-    (a, b) => a.startTime.getTime() - b.startTime.getTime(),
-  );
-
-  // 2. 겹치는 요소들을 그룹화
-  const groupedTasks: (T & {
-    ref: (node: HTMLElement | null) => void;
-    style: React.CSSProperties;
-  })[][] = [];
-
-  let currentGroup: typeof test = [];
-
-  sortedTasks.forEach((task) => {
-    if (currentGroup.length === 0) {
+function groupByOverlappingTimesRange<T extends BaseTask>(sortedTaskList: T[]) {
```

Comment:
```suggestion
function groupByOverlappingTimeRange<T extends BaseTask>(sortedTaskList: T[]) {
```
찾아보았는데 time은 s가 붙지 않는다구 하더라구요..!

Reply:
변경하였습니다!